### PR TITLE
html: removed left rails

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

- removed left panel in /profile, /toolbox, /new/funding
- replaces newletter and pushed it to the bottom

##### Checklist

- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Refers/Fixes
Fixes: https://github.com/gitcoinco/web/issues/411

### CODE REVIEW

**Comment by thelostone-mc** [comment]
@PixelantDesign  https://github.com/gitcoinco/web/issues/411#issuecomment-368020251

**Comment by thelostone-mc** [comment]
## Screenshots


### /profile

screenshot-2018-2-25 thelostone-mc gitcoin

### /funding/new

screenshot-2018-2-25 gitcoin push open source repos forward

### /toolbox

screenshot-2018-2-25 gitcoin toolbox

**Comment by thelostone-mc** [comment]
@owocki / @mbeacom  :  my submission didn't go through !!

Bounty: https://gitcoin.co/funding/details?url=https://github.com/gitcoinco/web/issues/411
Initially it gwei was set to be 0.1 gwei and that took too long

Initial submission (where the gwei was set to 0.1gwei by default ) : https://etherscan.io/tx/0x1762e7025519611b59924a6abf56c2acaf45cd299f346661c28620ee6d306cb2

Successful submission (where I raised it to 0.4 gwei ) : https://etherscan.io/tx/0xf0c3a85e9cbcab3d8f5fe824cc795e546a7707ba426cadae05a0543acd35fb5b

**Comment by owocki** [comment]
> It took close to 30min for my submission to turn up on the bounty.
> Worth creating an issue -> to see if it can be updated faster ?

unfortunately, i think this is just a consequence of working with the ethereum mainnet.  hopefully scaling / sharding makes things faster in the future!

**Comment by owocki** [comment]
@PixelantDesign did you want this to go live on the site?  do we want to remove the sidebar on ALL interior pages to the site, or just the ones you specified?

@thelostone-mc how confident do you feel that you've handled any responsiveness issues (mobile, tablet, etc)?

**Comment by thelostone-mc** [comment]
@owocki No issues rendering in mobile /tablet devices ! 
Just cause you asked  (I added a few extra lines off CSS to prettify the content on the mobile )
^That doesn't have anything to do with this PR but I added that anyways  -> Hope that's alright ^_^

**Comment by owocki** [comment]
makes sense to me @thelostone-mc 

@PixelantDesign let me know if youd like these changes live on the site and if you feel like they should apply to more interior pages than just the three specified in the desc ( /profile, /toolbox, /new/funding )

**Comment by PixelantDesign** [comment]
@owocki I think we can push it live.

**Comment by thelostone-mc** [comment]
@owocki all good!!

**Comment by owocki** [comment]
hey @thelostone-mc -- i was just chatting with @PixelantDesign on slack..  could you make sure that we've made this change to all interior pages ( tools, profile, leaderboard, submit funding, kill funding, accept funding … etc ) instead of just the ones we originally specified (  /profile, /toolbox, /new/funding )

sorry for the scope creep here.  it was a miscommunication internally.. hopefully should not be too much more work to do this while its still top of mind

**Comment by thelostone-mc** [comment]
@owocki Made the changes for everything except `leadeboard` as I'm having trouble populating data (what to where)


## Screenshots

### /process

screenshot-2018-2-27 gitcoin push open source repos forward

### /kill

screenshot-2018-2-27 gitcoin push open source repos forward 1


### /fulfill
screenshot-2018-2-27 gitcoin push open source repos forward 2

**Comment by thelostone-mc** [comment]
## Screenshots

### /leaderboard

screenshot-2018-2-28 leaderboard top payers


@PixelantDesign @owocki Done and dusted :D 

@owocki thanks for the data population # Need to read docs more carefully 😅

**Comment by PixelantDesign** [comment]
Looks great! Thanks @thelostone-mc!

**Comment by PixelantDesign** [comment]
@thelostone-mc I found one more screen! Can we please remove? Thanks!

screen shot 2018-03-01 at 10 33 20 am

File: app/dashboard/views.py [hunk 1]
```diff
@@ -575,6 +576,7 @@ def profile(request, handle):
     params = {
         'title': 'Profile',
         'active': 'profile_details',
+        'newsletter_headline': "Be the first to know about new funded issues.",
```
Comment (mbeacom) [inline app/dashboard/views.py]: Just a nitpick note...  Please be mindful of maintaining consistency with usage of `'` and `"`

Comment (thelostone-mc) [reply app/dashboard/views.py]: Ah my bad !!

Comment (mbeacom) [reply app/dashboard/views.py]: No worries - Thanks for the contribution!

**Comment by thelostone-mc** [comment]
# Screenshots

## /loading Page
screen shot 2018-03-02 at 11 38 46 am


@PixelantDesign @mbeacom changes done :)

**Comment by darkdarkdragon** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 0.001 ETH (0.69 USD @ $692.1/ETH) attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](http://localhost:8000/issue/gitcoinco/web/486/455).
 * Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $0 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Pull request merged**